NOTICE.txt

```diff
@@ -0,0 +1,16 @@
+====
```

Comment:
same, i just put the one coming from elasticsearch-cloud-kubernetes-2.4.0_01
ok sure